soc/arm/nuvoton_npcx/common/soc_dt.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _NUVOTON_NPCX_SOC_DT_H_
+#define _NUVOTON_NPCX_SOC_DT_H_
+
+/* Macro function used foe device tree */
+
+/******************************************************************************/
```

Comment:
Comment style should not have all those stars.

Just

/*
 * Devicetree macros...
 * ...
 */